app/controllers/ops_controller/settings/common.rb

```diff
@@ -351,26 +338,6 @@ def settings_update_save
       @edit[:new].set_worker_setting!(:MiqWebServiceWorker, :memory_threshold, human_size_to_rails_method(w[:memory_threshold]))
 
       @update = MiqServer.find(@sb[:selected_server_id]).get_config
-    when "settings_database"                                      # Database tab
```

Comment:
I think you missed settings_update_db_verify up above, line 247-262 above :point_up: